presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -89,6 +89,7 @@
 import com.facebook.presto.sql.planner.iterative.rule.TransformCorrelatedLateralJoinToJoin;
```

Comment:
Commit message is wrong. It should be `Migrate TransformCorrelatedSingleRowSubqueryToProject to IterativeOptimizer Rule`

Reply:
I worded the commit message so since this is not just a migration, but adding more functionality on top. I can reword the message to reflect both the changes here i.e. migration and supporting multiple projection queries.

Reply:
In general try to avoid `adding more`. Try to keep single commit per single change.

Reply:
Ok, will divide the PR